src/components/tree/tree-node/tree-node.ts

```diff
@@ -275,4 +275,8 @@ export class MTreeNode extends ModulVue {
     private get isCheckboxAutoSelect(): boolean {
         return this.checkboxes === MCheckboxes.WithCheckboxAutoSelect;
     }
+
+    private get hasRightIcon(): boolean {
+        return this.withCheckboxes && this.node.rightIconName !== undefined;
```

Comment:
tu aurais pas pu mettre juste !this.node.rightIconName? comme ça au moins on catch '' null et undefined d'un coup.

Reply:
Change pour !!